platforms/hyperledger-fabric/configuration/samples/network-proxy-none.yaml

```diff
@@ -176,6 +176,8 @@ network:
           consensus: raft
           grpc:
             port: 7050
+          metrics:
+            enabled: false
```

Comment:
Please mention the metrics port for the orderer and then use this port in the orderer chart.
Also update this port in the other network.yaml and update the network.yaml docs here docs/source/operations/fabric_networkyaml.md

Reply:
Updated the files and docs